helpers/git.js

```diff
@@ -32,12 +32,19 @@ var fs = require('fs');
  * @throws {Error} If cannot get last git commit
  */
 function getLastCommit() {
-	var spawn = childProcess.spawnSync('git', ['rev-parse', 'HEAD']);
-	var err = spawn.stderr.toString().trim();
+	let spawn;
 
-	// If there is git tool available and current directory is a git owned directory
-	if (!err) {
-		return spawn.stdout.toString().trim();
+	try {
+		spawn = childProcess.spawnSync('git', ['rev-parse', 'HEAD']);
+		// If there is git tool available and current directory is a git owned directory
+		if (!spawn.stderr.toString().trim()) {
```

Comment:
This still raises a `TypeError: Cannot read property 'toString' of null`. The issue persists.

Reply:
Yes that problem really exits. Easy to test is to run in node console. 

```
cpr.spawnSync('agit', ['rev-parse', 'HEAD']).stderr`
```